examples/tree_2d_dgsem/elixir_navier_stokes_source_terms.jl

```diff
@@ -0,0 +1,272 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the linear advection-diffusion equation
+
+equations = CompressibleEulerEquations2D(1.4)
+# Note: If you change the Navier-Stokes parameters here, also change them in the initial condition
+# I really do not like this structure but it should work for now
+equations_parabolic = CompressibleNavierStokes2D(1.4,  # gamma
+                                                 1000, # Reynolds number
+                                                 0.72, # Prandtl number
+                                                 0.5,  # free-stream Mach number
+                                                 1.0,  # thermal diffusivity
+                                                 equations)
```

Comment:
1. I am not sure if we need all these constants. For example, the Prandtl number basically never changes för the ideal gas assumption. I threw in all the constants I thought we would need and am open to trimming it down. Also, we could possibly store `mu = 1 / Reynolds` for the sake of code readability.
2. Keywords is probably the best way to do this but I wasn't sure if having the constructor look like
```julia
function CompressibleNavierStokes2D(equations; gamma, Reynolds, Prandtl, Mach_freestream, kappa)
```
would be kosher to have the call structure
```julia
equations_parabolic = CompressibleNavierStokes2D(gamma=1.4,
                                                 Reynolds=1000,
                                                 Prandtl=0.72,
                                                 Mach_freestream=0.5,
                                                 kappa=1.0,  # thermal diffusivity
                                                 equations)
```
for the sake of consistency with the `LaplaceEquation2D` where `equations` comes last.

Reply:
Keyword arguments need to come after positional ones, so it could be
```julia
CompressibleNavierStokes2D(equations; Reynolds, Prandtl, Mach_freestream, kappa)
```
where we should grab `gamma` from `equations`.

Reply:
@sloede We should probably unify the argument names with the lattice Boltzmann stuff (using `Ma` and `Re` for now).

Reply:
Done